docs/reference/admin/layouts/layout-kit.mdx

```diff
@@ -0,0 +1,199 @@
+---
+title: Layout Kit
+---
+
+import CodeBlock from '@theme/CodeBlock'
+import _LayoutUISource from '!!raw-loader!./_Example/my-custom-layout.tsx'
+import _components_LayoutSource from '!!raw-loader!./_Example/components/Layout.tsx'
+import _pages_example_Source from '!!raw-loader!./_Example/pages/example.tsx'
+
+# <span className="version">Interface 1.2+</span> LayoutKit
+
+Most application layouts are divided into panels, have a header and footer bars and
+in most cases, at least one sidebar.
+
+For this reason, Contember comes with a set of pre-built components that you can use to build your
```

Comment:
I would mention that the LayoutKit are UI components and what are they handling.

Reply:
Done